ajax/libs/particle-api-js/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "particle-api-js",
+  "filename": "particle.min.js",
+  "version": "5.3.0",
+  "description": "Particle API Client",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/spark/particle-api-js.git"
+  },
+  "author": {
+    "name": "Emily Rose",
+    "email": "nexxy@particle.io"
+  },
+  "keywords": [
+    "particle",
+    "library",
+    "spark",
+    "api"
+  ],
+  "license": "Apache-2.0",
```

Comment:
This can be. Next time you use difference info than origin package.json from the package you grabbed, in this case, npm, should better provide the reference first.